var/spack/repos/builtin/packages/py-pager/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPager(PythonPackage):
+    """Python module that pages output to the screen,
+       reads keys and console dimensions without executing external utils."""
+    url      = "py-pager"
```

Comment:
This isn't a real URL, you should replace this with the URL below

Reply:
fixed